submissions/Adebiyi_Osinibi/HTML/home.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
I like the page structure 👍 . However I'd be nice to see some elements of contrast on that page. It's really difficult to read white text on a lemon background